week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>I suggest to define accuracy (here or in the theory part (4. Theory)) - you probably do all this in the slides, so you could in the notebook maybe only add some kind of glossar for very important terms in ML?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>will do :)</p>